lib/Slackbot_worker.js

```diff
@@ -1,17 +1,21 @@
 var Ws = require('ws');
+var clone = require('lodash.clone');
```

Comment:
I don't see lodash in the dependencies. Would you rather have lodash instead of the modular clone method?